ecommerce/api.py

```diff
@@ -480,3 +482,28 @@ def calculate_run_price(course_run, user):
     coupon = coupons[0]
     price = calculate_coupon_price(coupon, price, course_run.edx_course_key)
     return price, coupon
+
+
+def validate_prices():
+    """
+    Validate prices and financial aid discounts
+    Returns:
+        (list): List of validation errors
+    """
+    errors = []
+    programs = Program.objects.filter(live=True)
+    for program in programs:
+        runs = CourseRun.objects.filter(course__program=program)
+        course_prices = [CoursePrice.objects.filter(course_run=run, is_valid=True).first() for run in runs]
+        prices = [course_price.price if course_price else None for course_price in course_prices]
+        if not all(prices[0] == price for price in prices):
+            errors.append('Invalid course prices for program {0}'.format(program.title))
+
+        if program.financial_aid_availability:
```

Comment:
I didn't say this in the issue but can you also verify that there is one `discount_amount` which is `0` and one `income_threshold` which is `0`?